src/embed_tests/pyinitialize.cs

```diff
@@ -24,10 +24,12 @@ public static void StartAndStopTwice()
         public static void LoadDefaultArgs()
         {
             using (new PythonEngine())
-            using (var argv = new PyList(Runtime.Runtime.PySys_GetObject("argv")))
             {
-                Runtime.Runtime.XIncref(argv.Handle);
-                Assert.AreNotEqual(0, argv.Length());
+                var argvref = new BorrowedReference(Runtime.Runtime.PySys_GetObject("argv"));
+                using(var argv = new PyList(argvref.DangerousGetAddress()))
```

Comment:
Rather than adding `DangerousGetAddress` here, we should have an `internal PyList(BorrowedReference)` constructor.